Blog/templates/Blog/homepage.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'Blog/base.html' %}
+{% block title %} ANONYMOUS BLOG {% endblock %}
+{% block header %}<b> ANONYMOUS BLOG</b> {% endblock %}
+{% block pagecontent %}
+    <form action="{% url 'Blog:homepage' %}" method="post" id="home_page">
+        {% csrf_token %}
+        <div id="Wrapper" style="padding-top : 25px;">
+                <div id="first">
```

Comment:
Use correct indentation. And please refer Django official documentation. Their section on displaying errors is https://docs.djangoproject.com/en/2.1/topics/forms/#rendering-form-error-messages

```
<div id="first">
  <label>Title:</label>
  <p>
    {{form.title}}
    {% if form.title.errors %}
      {{form.title.errors}}
    {% endif %}
  </p>
</div>
```